tox.ini

```diff
@@ -17,6 +17,7 @@ commands = sphinx-build -W -n -b html -d {envtmpdir}/doctrees source {envtmpdir}
 
 [testenv:ui]
 description = Run selenium tests
+deps = {[testenv]deps}
```

Comment:
Didn't realize `xvfbwrapper` was in `requirements.txt`, could you remove it and add it here as it appears below?
```
deps =
    xvfbwrapper
    {[testenv]deps}
```

Ideally testing dependencies are only installed during testing, thanks for fixing my oversight!

FYI `[testenv:ui]` inherits from `[testenv]`, so `deps = {[testenv]deps}` isn't required as it's already inherited :)